<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWindow</class>
 <widget class="QMainWindow" name="PlayerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="enterGameButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>进入游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnToMenuButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>400</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>返回主菜单</string>
    </property>
   </widget>
   <widget class="QLabel" name="selectplayerLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择游戏玩家：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="createNewPlayerLineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>305</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="selectplayerComboBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>205</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="createNewPlayerLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>新建游戏玩家：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeNameButton">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>255</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>修改名字</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteAllButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>600</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除全部玩家</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>255</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>删除玩家</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
